api/src/client/dynamics/fire-safety/DynamicsFireSafetyInspectionClient.ts

```diff
@@ -0,0 +1,60 @@
+import { FireSafetyInspection } from "@client/dynamics/fire-safety/types";
+import { LogWriterType } from "@libs/logWriter";
+import axios, { AxiosError } from "axios";
+
+export interface FireSafetyInspectionClient {
+  getFireSafetyInspectionsByAddress: (
+    accessToken: string,
+    address: string
+  ) => Promise<FireSafetyInspection[]>;
+}
+export const DynamicsFireSafetyInspectionClient = (
+  logWriter: LogWriterType,
+  orgUrl: string
+): FireSafetyInspectionClient => {
+  const getFireSafetyInspectionsByAddress = async (
```

Comment:
[pebble/sand] getFireSafetyInspection instead of getFireSafetyInspectionByAddress because address isn't the only parameter being passed in?